text-review/code.js

```diff
@@ -0,0 +1,24 @@
+const find = "figma";
+const suggestion = "✨Figma✨";
+const color = "GREEN";
+
+if (figma.command === "textreview") {
+  figma.on("textreview", async ({ text }) => {
+    const chunks = text
+      .replace(new RegExp(suggestion, "g"), suggestion.replace(/./g, "x"))
+      .toLowerCase()
+      .split(find);
+    let end = 0;
+    return chunks
+      .map((string, i) => {
+        if (i === chunks.length - 1) return null;
+        const start = end + string.length;
+        end = start + find.length;
+        return { start, end, suggestions: [suggestion], color };
+      })
+      .filter(Boolean);
+  });
+} else {
+  figma.notify("I am running like any other plugin!");
+  figma.closePlugin();
+}
```

Comment:
Is there a reason not to make this typescript?

Reply:
This is very similar to the one ChatGPT wrote lol https://www.figma.com/plugin-docs/textreview-plugins#a-quick-example

Reply:
I've been on the fence about TS for some of these. For one, it's easier to try this out when you can literally copy paste one file without worrying about a compilation step. 

If this was a starter, I'd lean into formalizing it a bit more. I keep seeing these as MVPs to demonstrate a concept, more than something you would download to start a project. idk. You suggesting TS is making me question my assumption.

Reply:
Yeah I think this is a good call